app/src/main/java/com/babylon/wallet/android/presentation/transfer/TransferViewModel.kt

```diff
@@ -477,6 +472,22 @@ sealed class TargetAccount {
     ) : TargetAccount() {
         override val address: String
             get() = account.address
+
+        fun hasAnyDenyDepositRule(forSpecificAssetAddress: String): Boolean {
```

Comment:
maybe this could be profile extension function on ThirdPartyDeposits?

Reply:
hmm yes makes absolutely sense!